modules/ztp-preparing-the-hub-cluster-for-ztp.adoc

```diff
@@ -0,0 +1,101 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/ztp-zero-touch-provisioning.adoc
+
+[id="ztp-preparing-the-hub-cluster-for-ztp_{context}"]
+= Preparing the hub cluster for ZTP
+
+Use these procedures to configure your hub cluster with a set of ArgoCD applications that generate the required installation and policy CRs for each site based on a zero touch provisioning (ZTP) GitOps flow.
```

Comment:
s/CRs/custom resources (CR)

Suggestion:
"You can configure your hub cluster with a set of ArgoCD applications that generate the required installation and policy custom resources (CR) for each site based on a zero touch provisioning (ZTP) GitOps flow."

Reply:
Are there any prerequisites for this one?

Reply:
changed